.travis.yml

```diff
@@ -103,8 +103,10 @@ jobs:
         - bundle install
         - gem install xcpretty
         - pod --version
-        - sed -i.bak -e s/use_frameworks/#use_frameworks/ Example/Podfile
-        - ./scripts/if_changed.sh bundle exec pod install --project-directory=Example --repo-update
+        - cd Example # Pre Firebase 5.0.0 Analytics modular header workaround requires cd
```

Comment:
Can these all be run in a single script stage (i.e. a single `-`)? It's really surprising to me that `cd` in one script has any effect on the subsequent ones.